### DISCUSSION TITLE
Trouble with Japanese Stable Diffusion XL

### DISCUSSION THREAD

**Discussion by mitkamata** [huggingface/diffusers#7673] [General]
I installed Japanese Stable Diffusion XL on my local PC (Windows 11).
In my environment, it seems that there is an error due to a mismatch in the arguments for "def check_inputs" starting from line 590 and "self.check_inputs" starting from line 1012 in "pipeline_stable_diffusion_xl.py".
After commenting out two arguments from "self.check_inputs", Japanese Stable Diffusion XL started working.

**Comment by tolgacangoz** [comment]
This is because they wrote their [`JapaneseStableDiffusionXLPipeline` with `diffusers==0.23.0`](https://huggingface.co/stabilityai/japanese-stable-diffusion-xl/blob/main/model_index.json#L2-L3).

**Reply by mitkamata** [reply]
I am using version 0.27.2. Thank you.
